### PR TITLE
Committing changes for the setup of dev environment

### PR DESCRIPTION
**Pull request by abhishekrao93**

### CODE REVIEW

**Comment by abhishekraotest** [comment]
Test

File: Setup Dev Environment/Colab File [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Please find the link below to the Colab Notebook for the Quality rating Bar Chart.
```
Comment (nabi-rezvani) [inline Setup Dev Environment/Colab File]: Please adjust this file and use markdowns
https://guides.github.com/features/mastering-markdown/

File: Setup Dev Environment/Readme.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Setup-development-environment
+The Repository is created to track the internship activities at ITIC
+
+
+TO DO :
+
+-Setup Python 3.6 or 3.7 on your machine.
```
Comment (nabi-rezvani) [inline Setup Dev Environment/Readme.md:7]: How? Add commands e.g. the following:
```
brew install --ignore-dependencies https://raw.githubusercontent.com/Homebrew/homebrew-core/f2a764ef944b1080be64bd88dca9a1d80130c558/Formula/python.rb
```

File: Setup Dev Environment/Colab File (hunk 1 above)
Comment (abhishekrao93) [reply Setup Dev Environment/Colab File]: I have converted the notepad file to a markdown file

File: Setup Dev Environment/Readme.md (hunk 2 above)
Comment (abhishekrao93) [reply Setup Dev Environment/Readme.md:7]: I installed python using the python executable file on my windows machine. I have updated the steps in  the document explaining how to install python on the machine

**Pull request merged**